packages/editor-ui/src/components/SettingsSidebar.vue

```diff
@@ -123,15 +121,26 @@ export default mixins(
 	ul {
 		height: 100%;
 	}
+
+	:global(.el-menu-item) > span{
+		position: relative;
+		left: 8px;
+	}
+}
+
+.returnButton :global(.n8n-heading) {
```

Comment:
could we do this without these global changes?

Reply:
Unfortunately, `n8n-heading` component doesn't propagate classes so cannot attach css class to it. The only other ways to avoid global class here that come to my mind are to target `span` tag or to wrap `n8n-heading` inside element and attach module class to it.
Going with the first approach.

Reply:
👍